tests/integration/test_celery.py

```diff
@@ -3,27 +3,42 @@
 
 from contextlib import contextmanager
 
-from celery import Celery
+import celery
+import pytest
+from celery.signals import setup_logging
 
 import scout_apm.celery
 from scout_apm.api import Config
 
+# http://docs.celeryproject.org/en/latest/userguide/testing.html#py-test
+skip_unless_celery_4_plus = pytest.mark.skipif(
+    celery.VERSION < (4, 0), reason="pytest fixtures added in Celery 4.0"
+)
+
+
+@setup_logging.connect
+def do_nothing(**kwargs):
+    # Just by connecting to this signal, we prevent Celery from setting up
+    # logging - and stop it from interfering with global state
+    # http://docs.celeryproject.org/en/v4.3.0/userguide/signals.html#setup-logging
+    pass
+
 
 @contextmanager
-def app_with_scout(config=None):
+def app_with_scout(app=None, config=None):
```

Comment:
How does changing this method signature affect current calls to this function? Who calls this and from where?